### PR TITLE
Add ability to specificy a custom page to redirect users with expired sessions

### PR DESCRIPTION
**Pull request by comandrei**
Adds an optional parameter passed to the JS expiration handler.
If this is specified, instead of reloading the page it will redirect the user to the page configured in the setup of the SessionSecurity object.

Feedback and suggestions are very welcome!

### CODE REVIEW

**Comment by jpic** [comment]
Sounds good to me !

File: session_security/static/session_security/script.js [hunk 1]
```diff
@@ -49,7 +50,12 @@ yourlabs.SessionSecurity.prototype = {
     // seconds.
     expire: function() {
         this.expired = true;
-        window.location.reload();
+        if (this.returnToUrl) {
```
Comment (jpic) [inline session_security/static/session_security/script.js]: shouldn't this be !== undefined ?

Sorry for the late comment, i just thought i'd re-read before releasing this ^^

**Comment by jpic** [comment]
Released in 2.3.1

**Comment by comandrei** [comment]
Thanks!

**Pull request merged**